mysys/my_largepage.c

```diff
@@ -188,127 +193,77 @@ int my_init_large_pages(my_bool super_large_pages)
   return 0;
 }
 
+
+#if defined(HAVE_MMAP) && !defined(_WIN32)
+/* Solaris for example has only MAP_ANON, FreeBSD has MAP_ANONYMOUS and
+MAP_ANON but MAP_ANONYMOUS is marked "for compatibility" */
+#if defined(MAP_ANONYMOUS)
+#define OS_MAP_ANON     MAP_ANONYMOUS
+#elif defined(MAP_ANON)
+#define OS_MAP_ANON     MAP_ANON
+#else
+#error unsupported mmap - no MAP_ANON{YMOUS}
+#endif
+#endif /* HAVE_MMAP && !_WIN32 */
+
 /*
```

Comment:
Should be double star. Not sure if anybody is actually using doxygen, but we still seem to follow these rules.